<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolBarEditorDialog</class>
 <widget class="QDialog" name="ToolBarEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Toolbars</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/logsim.png</normaloff>:/icons/logsim.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="toolBarLabel">
       <property name="text">
        <string>Toolbar:</string>
       </property>
       <property name="buddy">
        <cstring>toolBarComboBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="toolBarComboBox"/>
     </item>
     <item>
      <widget class="QToolButton" name="addToolBarButton">
       <property name="toolTip">
        <string>Add Toolbar</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/icons/list-add.png</normaloff>:/icons/list-add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="removeToolBarButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Delete Toolbar</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/icons/list-remove.png</normaloff>:/icons/list-remove.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="renameToolBarButton">
       <property name="toolTip">
        <string>Rename Toolbar</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/icons/edit-rename.png</normaloff>:/icons/edit-rename.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="availableActionsLabel">
       <property name="text">
        <string>Available actions:</string>
       </property>
       <property name="buddy">
        <cstring>availableActionsListWidget</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="currentActionsLabel">
       <property name="text">
        <string>Current actions:</string>
       </property>
       <property name="buddy">
        <cstring>currentActionsListWidget</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QListWidget" name="availableActionsListWidget"/>
     </item>
     <item row="2" column="2">
      <layout class="QVBoxLayout" name="currentActionsLayout">
       <item>
        <widget class="QListWidget" name="currentActionsListWidget"/>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QGridLayout" name="buttonsLayout">
       <item row="2" column="0">
        <widget class="QToolButton" name="removeButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="icon">
          <iconset resource="../resources/resources.qrc">
           <normaloff>:/icons/go-previous.png</normaloff>:/icons/go-previous.png</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QToolButton" name="addButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="icon">
          <iconset resource="../resources/resources.qrc">
           <normaloff>:/icons/go-next.png</normaloff>:/icons/go-next.png</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <spacer name="centerSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="2">
        <widget class="QToolButton" name="moveDownButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="icon">
          <iconset resource="../resources/resources.qrc">
           <normaloff>:/icons/go-down.png</normaloff>:/icons/go-down.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QToolButton" name="moveUpButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="icon">
          <iconset resource="../resources/resources.qrc">
           <normaloff>:/icons/go-up.png</normaloff>:/icons/go-up.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="upSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="2">
        <spacer name="downSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="visibleCheckBox">
     <property name="text">
      <string>Visible</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>toolBarComboBox</tabstop>
  <tabstop>addToolBarButton</tabstop>
  <tabstop>removeToolBarButton</tabstop>
  <tabstop>availableActionsListWidget</tabstop>
  <tabstop>moveUpButton</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>moveDownButton</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>currentActionsListWidget</tabstop>
  <tabstop>visibleCheckBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ToolBarEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ToolBarEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
